nexterp_stock_date/models/stock.py

```diff
@@ -127,29 +150,39 @@ class StockValuationLayer(models.Model):
     _inherit = "stock.valuation.layer"
     _log_access = False
 
-    create_date = fields.Datetime('Created on', index=True, readonly=True)
-    create_uid = fields.Many2one('res.users', 'Created by', index=True, readonly=True)
-    write_date = fields.Datetime('Last Updated on', index=True, readonly=True)
-    write_uid = fields.Many2one('res.users', 'Last Updated by', index=True, readonly=True)
+    create_date = fields.Datetime("Created on", index=True, readonly=True)
+    create_uid = fields.Many2one("res.users", "Created by", index=True, readonly=True)
+    write_date = fields.Datetime("Last Updated on", index=True, readonly=True)
+    write_uid = fields.Many2one(
+        "res.users", "Last Updated by", index=True, readonly=True
+    )
+    create_date_in_reality = fields.Datetime(
```

Comment:
Si asta il poti scoate, nu e cazul...write_date pastreaza data in care se creeza daca nu sunt alte modificari pe el.

Reply:
la creere write_date e tot val_date.
odoo e prea vast si oricum nici un client nu va putea fara vreun consultat.
eu sunt pt cod care este usor de inteles cu explicatii cu help pe campuri si sa explice schimbarea fata de standard. 
daca e vreo problema, omul intra pe obiect si vede direct care poate fi problema.

aici cu _log_access = false nu mai vezi campurile in view metadata.   

corect  trebuia facut un alt camp accounting_date si orodonat dupa el   - dar cred ca trebuiau modificate ceva rapoarte

Reply:
da,dar nu mai mergea pentru alte localizari care folosesc create_date si run_fifo...deci daca chiar vrei sa tii campul, redenumeste-l si pe asta date_done ca sa fim consecventi cu celelalte tabele din stock.